### PR TITLE
Component merge: ToggleSlider has been merged with ToggleSwitch for mobile

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Problem] There is naming convention issue in mobile and warable
 profile with ToggleSlider and ToggleSwitch
[Solutuion]
+ new ToggleSwitch Slider style has been added
+ added feature "new" for styles

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

**Comment by lslachciak** [comment]
/home/travis/build/Samsung/TAU-Design-Editor/design-editor/src/panel/property/attribute/attribute-element-styles.js

   74:1  error  Expected indentation of 3 tabs but found 12 spaces  indent

   75:1  error  Expected indentation of 3 tabs but found 12 spaces  indent

   91:1  error  Expected indentation of 4 tabs but found 16 spaces  indent

   92:1  error  Expected indentation of 5 tabs but found 20 spaces  indent

   93:1  error  Expected indentation of 4 tabs but found 16 spaces  indent

  100:1  error  Expected indentation of 4 tabs but found 16 spaces  indent

✖ 6 problems (6 errors, 0 warnings)

  71 errors and 0 warnings potentially fixable with the `--fix` option.

File: tau-component-packages/components/toggleswitch-mobile/package.json [hunk 1]
```diff
@@ -30,5 +30,18 @@
   "draggable": true,
   "class": [
     "ui-toggleswitch-large"
+  ],
+  "styles": [
+    {
+      "icon": "./styles/toggleswitch-circle.png",
+      "name": "Toggle Switch",
```
Comment (lslachciak) [inline tau-component-packages/components/toggleswitch-mobile/package.json]: I woul call this style just "switch"

File: tau-component-packages/components/toggleswitch-mobile/package.json [hunk 2]
```diff
@@ -30,5 +30,18 @@
...
+      "template": "<input type=\"checkbox\" class=\"ui-toggleswitch\" />"
+    }, {
+      "icon": "./styles/toggleswitch-slider.png",
+      "name": "Toggle Switch (Slider)",
```
Comment (lslachciak) [inline tau-component-packages/components/toggleswitch-mobile/package.json]: I would call this style just slider

File: tau-component-packages/components/toggleswitch-mobile/package.json (hunk 1 above)
Comment (TomaszLukawskiSam) [reply tau-component-packages/components/toggleswitch-mobile/package.json]: Done

File: tau-component-packages/components/toggleswitch-mobile/package.json (hunk 2 above)
Comment (TomaszLukawskiSam) [reply tau-component-packages/components/toggleswitch-mobile/package.json]: Done

**Comment by TomaszLukawskiSam** [comment]
> general comment
> "toggleslider element still visible on the widgets list, lease remove it

Done

**Pull request merged**